### PR TITLE
Revise StatefulSet tutorial updates advice

### PR DESCRIPTION
**Pull request by sftim**
Split out from PR #42767

Revise the  “Updating StatefulSets” section of the [StatefulSet tutorial](https://k8s.io/docs/tutorials/stateful-application/basic-stateful-set).

### CODE REVIEW

**Review by PranitRout07 — COMMENTED** [review]
Hey @sftim  in line 564 the shell script is missing .

File: content/en/docs/tutorials/stateful-application/basic-stateful-set.md [hunk 1]
```diff
@@ -1114,14 +1143,11 @@ statefulset "web" deleted
...
-`.spec.podManagementPolicy` to the StatefulSet API Object.
-
-### OrderedReady Pod management
+identity.
 
-`OrderedReady` pod management is the default for StatefulSets. It tells the
-StatefulSet controller to respect the ordering guarantees demonstrated
-above.
+You can specify a Pod management policy to avoid this strict ordering;
+either [`OrdereReady`](/docs/concepts/workloads/controllers/statefulset/#orderedready-pod-management) (the default)
```
Comment (kbhawkey) [inline content/en/docs/tutorials/stateful-application/basic-stateful-set.md:1149]: Fix typo, `OrderedReady`

**Comment by sftim** [comment]
> Hey @sftim in line 564 the shell script is missing .

> You can split updates to a StatefulSet that uses the `RollingUpdate` strategy
> into _partitions_, by specifying `.spec.updateStrategy.rollingUpdate.partition`.
> You'll practice that later in this tutorial.

By “You'll practice that later in this tutorial”, what I mean to convey is that the reader must continue reading the tutorial to learn about this. Hence, no shell commands at this point. Instead, the reader should keep reading.

**Comment by PranitRout07** [comment]
> > Hey @sftim in line 564 the shell script is missing .
> 
> > You can split updates to a StatefulSet that uses the `RollingUpdate` strategy
> > into _partitions_, by specifying `.spec.updateStrategy.rollingUpdate.partition`.
> > You'll practice that later in this tutorial.
> 
> By “You'll practice that later in this tutorial”, what I mean to convey is that the reader must continue reading the tutorial to learn about this. Hence, no shell commands at this point. Instead, the reader should keep reading.

ok @sftim .

File: content/en/docs/tutorials/stateful-application/basic-stateful-set.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/tutorials/stateful-application/basic-stateful-set.md]: (fixed)

**Comment by sftim** [comment]
Relevant to https://github.com/kubernetes/website/issues/37203

**Review by dipesh-rawat — COMMENTED** [review]
Thanks!

/lgtm

**Pull request merged**